roles/openshift_router/tasks/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+
+# TODO: Centralized openshift_config openshift_facts??
+# TODO: anchor this on a master
+- name: Query for OpenShift router
+  command: openshift ex router --credentials={{ openshift_config }}
```

Comment:
We need to specify the --images flag for all installations except origin, since the generated config defaults to he origin image name: https://github.com/openshift/training/blob/master/beta-3-setup.md#creating-the-router